packages/reports/tests/acceptance/multi-datasource-test.js

```diff
@@ -67,10 +67,13 @@ module('Acceptance | multi-datasource report builder', function(hooks) {
     assert
       .dom('.report-builder__container--filters--collapsed')
       .containsText(
-        'Filters Container (id) equals 1 Date Time (day) between (<=>) 11/04/2020 and 11/05/2020 Used Amount greater than (>) 30',
+        'Filters Container (id) equals 1 Date Time (day) between (<=>)',
```

Comment:
Are these dynamic dates? Can we calculate the expected values in the test?

Reply:
The dates will update to be the 4th and 5th of whatever month it is, so if we have a tool to calculate that then we likely could

Reply:
We can do something like `moment().startOf('month').add(4, 'days').format('MM/DD/YYY')` but it'll change format from my PR

Reply:
yeah thats true, wanna just leave it this way for now and update it when your pr comes through to be more dynamic?